lib/charms/hpc_libs/v0/slurm_ops.py

```diff
@@ -26,14 +26,13 @@
 the inheriting Slurm service manager how to manage its corresponding Slurm service on the host.
 
 ```python3
-import charms.hpc_libs.v0.slurm_ops as slurm
-from charms.hpc_libs.v0.slurm_ops import SlurmManagerBase, ServiceType
+from charms.hpc_libs.v0.slurm_ops import SlurmManagerBase, ServiceType, SnapManager,
```

Comment:
IIRC we don't need SnapManager here now:

```suggestion
from charms.hpc_libs.v0.slurm_ops import SlurmManagerBase, ServiceType
```